libs/util/http.client.js

```diff
@@ -161,7 +163,9 @@ function io(url, options) {
         method: options.method || METHOD_GET,
         timeout: options.timeout,
         headers: options.headers,
-        body: options.data
+        body: options.data,
+        cors: options.cors,
+        useXDR: options.useXDR
```

Comment:
Looks like `cors` and `useXDR` serve the same purpose for the `xhr` module. `useXDR` is documented while `cors` isn't, so I would use `useXDR` only.